config_defaults.yml

```
@@ -36,7 +36,8 @@ Gemfile:
       - gem: puppet-strings
         version: '>= 2.2'
 Rakefile:
-  puppet_strings_patterns: []
+  config.user: voxpupuli
```

We shouldn't set the user explicitly. Instead, we derive it using modulesync (since 2e2ae76b60e53ecd7b4b5ebc9dad03f0b05bd32a & 8d5bca5a75d336e0e3c3286867f694430d4b7afd)
If you want to document them:
```suggestion
  # config.user: USER
  # config.project: PROJECT
```